<launch>
  <node pkg="serial_ros" type="serial_ros" name="serial_ros_node" output="screen">
    <param name="Serial_Port" value="/dev/ttyACM0"/>
    <param name="Baund_rate" value="115200"/>
    <param name="character_size" value="8"/>
    <param name="stop_bits" value="1"/>
    <param name="parity" value="2"/>
    <param name="flow_control" value="1"/>
  </node>
</launch>